src/applications/check-in/utils/navigation/index.js

```diff
@@ -86,4 +86,17 @@ const URLS = Object.freeze({
   TRAVEL_MILEAGE: 'travel-mileage',
 });
 
-export { updateFormPages, URLS };
+const isAnInternalPage = location => {
+  let valid = false;
+  const url = location.split('/');
+  // Analyze the part of the URL where internal pages are.
+  const pageFromUrl = url[3];
```

Comment:
This is nitpicky but is it possible to get the current path from the router rather than having to rely on the current URL structure?

Reply:
no, because the router doesn't exist yet when we need this. :(